main.js

```diff
@@ -1,89 +1,107 @@
+var secondsInput = document.getElementById("seconds");
 var accomplishInput = document.querySelector('#descriptionInput');
-var categoryError = document.getElementById('categoryError');
-var descriptionError = document.getElementById('descriptionError');
+//errors
 var minutesError = document.getElementById('minutesError');
 var secondsError = document.getElementById('secondsError');
+var descriptionError = document.getElementById('descriptionError');
+var categoryError = document.getElementById('categoryError');
+//timer, cards, etc
+var minCountdown = document.getElementById('minutesCountdown');
+var secCountdown = document.getElementById('secondsCountdown');
 var leftSection = document.getElementById('leftSection');
 var timerDisplay = document.getElementById('timerDisplay');
 var activityForm = document.getElementById('activityForm');
 var activity = document.getElementById('newActivity');
-var minCountdown = document.getElementById('minutesCountdown');
-var secCountdown = document.getElementById('secondsCountdown');
 var descriptionTitle = document.getElementById('descriptionTitle');
 var cardsContainer = document.getElementById('cardContainer');
-var newActivitySection = document.getElementById('newActivitySection')
-var createNewActivityButton = document.getElementById('createNewButton');
-
+var newActivitySection = document.getElementById('newActivitySection');
+//global variables
 var loggedActivities = [];
 var currentActivity;
 
 ////////// EVENT LISTENERS ///////////////
+window.onload = renderCard();
 startActivityButton.addEventListener('click', startActivity);
 startTimerButton.addEventListener('click', countdown);
 logActivityButton.addEventListener('click', logActivity);
 createNewActivityButton.addEventListener('click', goHome);
-window.addEventListener('load', renderCard);
 
-minutesInput.addEventListener("keypress", function (event) {
+minutesInput.addEventListener("keypress", function(event) {
   if (event.which != 8 && event.which != 0 && event.which < 48 || event.which > 57) {
     event.preventDefault();
   }
 });
 
-secondsInput.addEventListener("keypress", function (event) {
+secondsInput.addEventListener("keypress", function(event) {
   if (event.which != 8 && event.which != 0 && event.which < 48 || event.which > 57) {
     event.preventDefault();
   }
 });
 
-///////////// EVENT HANDLERS & FUNCTIONS ///////////////
-for(var i = 0; i < allButtons.length ; i ++) {
+for (var i = 0; i < allButtons.length; i++) {
   allButtons[i].addEventListener('click', selectCategory)
 }
 
+///////////// EVENT HANDLERS & FUNCTIONS ///////////////
+
 function selectCategory() {
   if (studyButton.checked) {
-    showElement(studyImgActive);
-    hideElement(studyImg)
-    hideElement(meditateImgActive)
-    showElement(meditateImg)
-    hideElement(exerciseImgActive)
-    showElement(exerciseImg)
-    startTimerButton.className = 'start-button circle-outline-Study';
+    studyButtonActive();
```

Comment:
Love this refactor utilizing helper functions! There might be an even dryer way to refactor this using elements as an argument.